### ISSUE TITLE
Crash since Hotfix 4.2.3

### ISSUE THREAD

**Issue by Buchnekromant** [msolefonte/vco#326]
It looks like the new 4.2.3 hotfix broke the mod; 

![image](https://github.com/msolefonte/vco/assets/7038110/4f4513ca-bd3b-4447-9d71-58a878fdcc6e)

**Comment by 0v0-Poi** [comment]
Yeah, I meet the same situation like that, tasted with this only mod active and met the same problem
